app/build.gradle

```diff
@@ -7,8 +7,8 @@ android {
         applicationId "io.mrarm.irc"
         minSdkVersion 17
         targetSdkVersion 30
-        versionCode 12
-        versionName "0.5.4"
+        versionCode 18
```

Comment:
Please do not change the version numbers in a pull request; pull requests are merged to the master branch and do not mean a new release; the code will be tagged and versioned manually.